rero_ils/modules/libraries/jsonschemas/libraries/library-v0.0.1.json

```diff
@@ -371,6 +371,134 @@
           }
         ]
       }
+    },
+    "acquisition_settings": {
+      "additionalProperties": false,
+      "type": "object",
+      "required": [
+        "shipping_informations"
+      ],
+      "properties": {
+        "shipping_informations": {
+          "title": "Shipping informations",
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+            "name",
+            "email",
+            "address"
+          ],
+          "properties": {
```

Comment:
We have multiple times the same definitions for an address. Would be nice to work with an $ref to an address definition.

Reply:
I tried to use a `$ref` for all properties, but it's not possible. `properties` key seems required a key/pair object and doesn't support a `$ref`. The full "acquisition information" block cannot be duplicate because, some required fields are not the same (email is required for shipping, not for billing). 

I updated the JsonSchema the best I could. I added some unitest and run a "bootstrap/setup" on localhost. All works

Reply:
It should be possible to work with `any_of` to resolve this, but maybe to complicated at the moment.